docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Concept Glossary
+----------------------------
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
```

Comment:
Hmm, I'd say it's less about understanding the sociotechnical context than about something like "relevant to help data scientists test assumptions made when measuring unobservable phenomena such as fairness." Or something along those lines

Reply:
I love this!